parlai/tasks/wizard_of_wikipedia/agents.py

```diff
@@ -467,6 +475,10 @@ def get(self, episode_idx, entry_idx=0):
             a['knowledge'] = (
                 TOKEN_NOCHOSEN + ' ' + TOKEN_KNOWLEDGE + ' ' + TOKEN_NOCHOSEN
             )
+        elif self.prepend_gold_knowledge:
+            a['text'] = '{} {} {}\n{}'.format(
+                TOKEN_KNOWLEDGE, a['checked_sentence'], TOKEN_END_KNOWLEDGE, a['text']
```

Comment:
we had used another token in wizard, https://github.com/facebookresearch/ParlAI/blob/19ab81fe9147ba17ef75942fbd7a94ec8dc8cb71/projects/wizard_of_wikipedia/generator/agents.py#L23